apps/opencs/model/prefs/state.cpp

```diff
@@ -166,6 +166,16 @@ void CSMPrefs::State::declare()
         setTooltip ("When using next/previous functions at the last/first item of a "
         "list go to the first/last item");
 
+    declareCategory ("Marker Size");
```

Comment:
I think it would be better to call this category "3D Scene Marker". We have several small categories that deal with random scene options. Perhaps these should all be consolidated?